web/src/lib/components/faces-page/merge-suggestion-modal.svelte

```diff
@@ -8,27 +8,57 @@
   import { mdiArrowLeft, mdiClose, mdiMerge } from '@mdi/js';
   import Icon from '$lib/components/elements/icon.svelte';
 
-  const dispatch = createEventDispatcher<{
-    reject: void;
-    confirm: [PersonResponseDto, PersonResponseDto];
-    close: void;
-  }>();
+  import { assetViewingStore } from '$lib/stores/asset-viewing.store';
 
   export let personMerge1: PersonResponseDto;
   export let personMerge2: PersonResponseDto;
   export let potentialMergePeople: PersonResponseDto[];
 
+  let { isViewing: showAssetViewer } = assetViewingStore;
   let choosePersonToMerge = false;
+  let changeFocus = false;
+  let buttonNo: HTMLButtonElement;
+  let buttonYes: HTMLButtonElement;
 
   const title = personMerge2.name;
 
+  const dispatch = createEventDispatcher<{
+    reject: void;
+    confirm: [PersonResponseDto, PersonResponseDto];
+    close: void;
+  }>();
+
+  const handleKeyboardPress = (event: KeyboardEvent) => {
+    if (!$showAssetViewer) {
+      event.stopPropagation();
```

Comment:
Why are you stopping propagation, especially if you aren't handling the keyboard event?